reference/api/settings.mdx

```diff
@@ -1910,3 +1911,189 @@ Reset an index's typo tolerance settings to their [default value](#typo-toleranc
+| **`model`**            | String           | Empty                                                                    | The model your embedder uses when generating vectors                                                                                                           |
+| **`documentTemplate`** | String           | `{% for field in fields %}{{field.name}}: {{field.value}}\n{% endfor %}` | Template defining the data Meilisearch sends the embedder                                                                                                      |
+| **`dimensions`**       | Integer          | Empty                                                                    | Number of dimensions in the chosen model. If not supplied, Meilisearch tries to infer this value                                                               |
+| **`revision`**         | String           | Empty                                                                    | Model revision hash                                                                                                                                            |
+| **`inputField`**       | Array of strings | []                                                                       | Path to location of document data in query Meilisearch sends to embedder                                                                                       |
+| **`inputType`**        | String           | `text`                                                                   | Type of data Meilisearch sends to embedder. Must be `text` or `textArray`                                                                                      |
+| **`query`**            | String           | Object                                                                   | Extra fields Meilisearch adds to queries Meilisearch sends to embedder                                                                                         |
+| **`pathToEmbeddings`** | Array            | []                                                                       | Path to vector embedding data in embedder response                                                                                                             |
+| **`embeddingObject`**  | String           | {}                                                                       | Name of the embedding object in embedder response                                                                                                              |
+| **`distribution`**     | Object           | Empty                                                                    | Describes the natural distribution of search results. Must contain two fields, `mean` and `sigma`, each containing a numeric value between `0` and `1`         |
+
+Depending on your chosen embedder, some of these fields may be mandatory.
```

Comment:
Depending on the chosen source, some of the fields may be mandatory, and only some of the fields are allowed.

I understand this may deviate from the usual presentation, but I would expect the reference to list which fields are allowed for each embedder source, and which fields are mandatory for each embedder source. Could be an addition to the table, or could be one table per source (as the exact meaning of parameters may change depending on the source)

Reply:
Yeah, the embedders settings are sort of stretching the format. I'll follow your suggestion.